coordinator/ethereum/test-utils/src/main/kotlin/net/consensys/zkevm/ethereum/MakefileContractDeploymentHelper.kt

```diff
@@ -85,7 +85,7 @@ fun makeDeployLineaRollup(
   deploymentPrivateKey?.let { env["DEPLOYMENT_PRIVATE_KEY"] = it }
   val command = when (contractVersion) {
     LineaContractVersion.V6 -> "make deploy-linea-rollup-v6"
-    else -> throw IllegalArgumentException("Unsupported contract version: $contractVersion")
+    // else -> throw IllegalArgumentException("Unsupported contract version: $contractVersion")
```

Comment:
This will trigger compilation error as `LineaContractVersion.V6` is currently the only possible value, comment out for now.